src/components/TeamPics/teampics.scss

```diff
@@ -0,0 +1,74 @@
+h1, h2, h3, h4, p {
```

Comment:
make the whole thing under Component since that is what you first called on your div from teampics.tsx and since we are using scss when you call h1, h2, etc headers it wont affect the whole other components/pages once everything is under the main div, it would only affect yours. Please look at the rules of scss or look at the title component to see how I did it.

Reply:
Changed, with all css code under <body></body>